### PR TITLE
Cleanup geant4e propagator

### PR DESCRIPTION
**Pull request by civanch**
#### PR description:
Removed misleading printout at initialisation of Geant4e propagator. Results of propagation should be unchanged.

#### PR validation:
private

### CODE REVIEW

File: TrackPropagation/Geant4e/src/Geant4ePropagator.cc [hunk 1]
```diff
@@ -312,18 +310,11 @@ std::pair<TrajectoryStateOnSurface, double> Geant4ePropagator::propagateGeneric(
 
   theG4eManager->InitTrackPropagation();
 
-  // re-initialize navigator to avoid mismatches and/or segfaults
-  theG4eManager->GetErrorPropagationNavigator()->LocateGlobalPointAndSetup(
-      g4InitPos, &g4InitMom, /*pRelativeSearch = */ false, /*ignoreDirection = */ false);
```
Comment (mmusich) [inline TrackPropagation/Geant4e/src/Geant4ePropagator.cc:317]: this should not be removed.

File: TrackPropagation/Geant4e/src/Geant4ePropagator.cc [hunk 2]
```diff
@@ -312,18 +310,11 @@ std::pair<TrajectoryStateOnSurface, double> Geant4ePropagator::propagateGeneric(
...
-
   bool continuePropagation = true;
   while (continuePropagation) {
     iterations++;
     LogDebug("Geant4e") << std::endl << "step count " << iterations << " step length " << finalPathLength;
 
-    // re-initialize navigator to avoid mismatches and/or segfaults
-    theG4eManager->GetErrorPropagationNavigator()->LocateGlobalPointWithinVolume(g4eTrajState.GetPosition());
```
Comment (mmusich) [inline TrackPropagation/Geant4e/src/Geant4ePropagator.cc:325]: this should not be removed.

Comment (civanch) [reply TrackPropagation/Geant4e/src/Geant4ePropagator.cc:325]: It was mistake to make modifications on top of 12_4_12, now it is done on top of master.

**Comment by civanch** [comment]
unlold

**Comment by civanch** [comment]
unhold

File: TrackPropagation/Geant4e/src/Geant4ePropagator.cc (hunk 1 above)
Comment (civanch) [reply TrackPropagation/Geant4e/src/Geant4ePropagator.cc]: Done

**Comment by civanch** [comment]
please test

**Comment by civanch** [comment]
+1

**Comment by perrotta** [comment]
+1

**Pull request merged**